installers/macos-pkg-setup-template.sh

```diff
@@ -54,7 +54,13 @@ echo "Create additional symlinks (Required for the UsePythonVersion Azure Pipeli
 ln -s ./bin/$PYTHON_MAJOR_DOT_MINOR python
 
 cd bin/
-ln -s $PYTHON_MAJOR_DOT_MINOR $PYTHON_MAJOR_MINOR
+
+# This symlink already exists if Python version with the same major.minor version is already installed, 
```

Comment:
This symlink already exists if Python version with the same major.minor version is installed,

Reply:
```suggestion
# This symlink already exists if Python version with the same major.minor version is already installed, 
```